jwtverifier.go

```diff
@@ -222,7 +222,23 @@ func (j *JwtVerifier) validateAudience(audience interface{}) error {
 func (j *JwtVerifier) validateClientId(clientId interface{}) error {
 	// Client Id can be optional, it will be validated if it is present in the ClaimsToValidate array
 	if cid, exists := j.ClaimsToValidate["cid"]; exists && clientId != cid {
-		return fmt.Errorf("clientId: %s does not match %s", clientId, cid)
+
+		switch v := clientId.(type) {
```

Comment:
Curious since I'm not familiar with Golang's patterns, is it common to use this naming convention? Asking for `v` specifically.

Reply:
ya.  `v` for `value` :)